openlibrary/components/MergeUI.vue

```diff
@@ -40,8 +46,19 @@ export default {
 }
 </script>
 
-<style>
+<style lang="less">
 #app {
-  font-family: Roboto;
+    font-size: 0.9em;
+
+    button {
```

Comment:
Limit this to only affect buttons in this given form
```suggestion
    & > button {
```